Src/AutoFixture/Kernel/OmitRecursionGuard.cs

```diff
@@ -1,34 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Collections;
-
-namespace Ploeh.AutoFixture.Kernel
-{
-    public class OmitRecursionGuard : RecursionGuard
```

Comment:
I'm sorry, I wasn't even aware that this file wasn't in the project file. It looks like I made a mistake when merging back in ca84aba8742027bf9bb10ddd7893e39b0d4d5b08, where it seems that I accidentally added it (again), possibly among other files that also ought not to be there.